reports/report_csv.py

```diff
@@ -0,0 +1,174 @@
+import csv, os
+from .report_details import *  # noqa
+from forms.modelutils import (TOPICS, FUNCTION, YESNO, )
+
+def get_gender(gender_id):
+    if gender_id == 1:
+        return "Female"
+    elif gender_id == 2:
+        return "Male"
+    elif gender_id == 3:
+        return "Transgender"
+    else:
+        return "Don't Know"
+
+def ws_05_csv(writer, counts_list, row, **kwargs):
+    for media_type in row:
+        for topic in row[media_type]:
+            topic_name = [x for x in MAJOR_TOPICS if x[0] == topic[1]][0][1]
+            gender = get_gender(topic[0])
+
+            writer.writerow({'Topic': topic_name, 'Gender': gender, 'Medium': media_type, 'Count': row[media_type][topic]})
```

Comment:
```suggestion
            gender = get_gender(topic[0])
            writer.writerow({'Topic': topic_name, 'Gender': gender, 'Medium': media_type, 'Count': row[media_type][topic]})
```

Just to avoid any indentation errors.